spec/features/clusters/new_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe "cluster create", type: :feature do
+  describe "create page" do
+    it "curated cluster creation" do
+      user = create(:user)
+
+      visit "/users/#{user.username}/curated_clusters"
+      click_link "New Curated Cluster"
+      expect(page).to have_current_path("/users/#{user.username}/curated_clusters/new", ignore_query: true)
+
+      fill_in :name, with: "Matt's Cluster"
+      click_on "Create Curated Cluster"
+
+      expect(page).to have_current_path("/users/#{user.username}/curated_clusters")
```

Comment:
why is "have_current_path" proper nomenclature? I haven't seen that before.

Reply:
When attempting to commit, rubocop had some issues with the shoulda-matchers from before. The new syntax is "have_current_path" for verifying the page you should be on.